doc/release-notes/10889_bump_PG17_FlyWay10.md

```diff
@@ -0,0 +1,3 @@
+This release bumps both the Postgres JDBC driver and Flyway versions. This should better support Postgres version 17, and as of version 10 Flyway no longer requires a paid subscription to support older versions of Postgres.
```

Comment:
At https://guides.dataverse.org/en/6.4/installation/prerequisites.html#postgresql we say, "PostgreSQL 13 is recommended because it’s the version we test against. Version 10 or higher is required because that’s what’s [supported by Flyway](https://documentation.red-gate.com/fd/postgresql-184127604.html), which we use for database migrations."

Is that still accurate. One we merge this is version 10 or higher of Postgres still what's supported by Flyway?

Reply:
correct, this is still accurate.